pdfplumber/display.py

```diff
@@ -32,13 +36,14 @@ def get_page_image(pdf_path, page_no, resolution):
             return converted
 
 class PageImage(object):
-    def __init__(self, page, original=None, resolution=DEFAULT_RESOLUTION):
+    def __init__(self, page, remove_alpha=True, original=None, resolution=DEFAULT_RESOLUTION):
```

Comment:
Is the current behavior to create an image with the alpha channel included? If so, should the default be `False`?